testsuite/tests/kuadrant/authorino/dinosaur/test_dinosaur.py

```diff
@@ -8,7 +8,7 @@
 pytestmark = [
     pytest.mark.authorino,
     pytest.mark.xfail(
-        reason="AuthPolicy max limit - https://github.com/Kuadrant/testsuite/issues/378",
+        reason="AuthPolicy max limit - https://github.com/Kuadrant/kuadrant-operator/issues/566",
```

Comment:
It should be marked by python.mark.issue as in other tests

Reply:
Addressed